ui/app/templates/components/clients/attribution.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="chart-wrapper dual-chart-grid">
-  <div class="chart-header has-export">
+  <div class="chart-header has-export has-bottom-margin-m">
```

Comment:
using sizing specific margin classes (from `helpers.scss`) so each template header has the correct margin size